app/models/order.rb

```diff
@@ -0,0 +1,9 @@
+class Order < ApplicationRecord
+  belongs_to :user
+  belongs_to :apartment
+
+  enum status: { booked: 0, in_progress: 1, cancelled: 2, expired: 3 }
+
+  validates :total, :start_from, :end_at, :user_id, :apartment_id, presence: true
+  validates :total, numericality: { greater_than: 0 }
```

Comment:
on the 7 line is enumeration of fields which has to be presensed
on the 8 line only fields which have to be more than 0 so it's not the same